MCprep_addon/MCprep_resources/mcprep_data_update.json

```diff
@@ -4525,10 +4632,5 @@
 		"sunflower",
 		"water_still",
 		"water"
-	],
```

Comment:
Just calling out that the original problem was that this structure was removed entirely, since I gather it was at one point manually edited in a commit. Now, it is just moved up above since it's generated from the base mcprep json file.